modules/policy-identity-access-management.adoc

```diff
@@ -49,8 +49,10 @@ In AWS, SREs generate a short-lived AWS access token for the `BYOCAdminAccess` u
 
 In Google Cloud, SREs access resources after being authenticated against a Red Hat SAML identity provider (IDP). The IDP authorizes tokens that have time-to-live expirations. The issuance of the token is auditable by corporate Red Hat IT and linked back to an individual user.
 
+include::osd-sre-sts-view-aws-account.adoc[leveloffset=+1]
```

Comment:
@smunje1 , you can't include a module in a module.

Reply:
If you don't need the heading but do need it in the middle of the module, consider transforming it into into a [snippet](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#writing-text-snippets).

If you aren't expecting to use it in more than one location, include the text directly.

Reply:
Ack. Thank you!